### DISCUSSION TITLE
Create Windows executables (`.exe`) from a PyVistaQt application

### DISCUSSION THREAD

**Discussion by banesullivan** [pyvista/pyvista#5097] [Announcements]
All credit goes to @akaszynski on this one!

Here is a template repository for creating a standalone PyVistaQt application using Pyinstaller. It even has CI workflows to automatically build the Windows executable!

cc @pyvista/pyvistaqt-admin 

https://github.com/pyvista/pyvistaqt-exe

**Comment by banesullivan** [comment]
I wish I had the time to do the same with Trame instead of Qt (for both Windows and Mac) too. 

Maybe @jourdain would want to help 😄 ?

**Comment by jourdain** [comment]
Indeed, doing a standalone binary/app via trame seems to be a good idea.
